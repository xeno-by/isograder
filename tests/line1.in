* Section [/1]

-1: This is a really huge mistake
that deserves a multi-line explanation.
