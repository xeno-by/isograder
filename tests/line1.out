EXIT CODE:
1

STDOUT:

STDERR:
error parsing the correction sheet:
line 4: invalid line: should either start with * (section) or start with -/+ (correction) or end with - (header)
NOTE: by the way, multi-line explanations for corrections are not supported
that deserves a multi-line explanation.
^

